app/src/main/java/com/mlinde/marvelcomicsapp/api/ApiRensponse.kt

```diff
@@ -1,6 +1,6 @@
 package com.mlinde.marvelcomicsapp.api
 
-sealed class ApiRensponse{
-    class Success(val data: Any?): ApiRensponse()
-    class Error(val message: String): ApiRensponse()
+sealed class ApiRensponse<out R>{
```

Comment:
Why` <out R>`? In `class Success` you're using `<out T>`. This should be the same object type.